evm_loader/program/src/precompile_contracts.rs

```diff
@@ -354,7 +354,7 @@ pub fn query_account<'a, B: AccountStorage>(
             let executable = state.query_solana_account_executable(account_address);
             if let Some(executable) = executable {
                 debug_print!("query_account executable result: {}", executable);
-                let executable: U256 = (executable as u8).into(); // pad to 32 bytes
+                let executable: U256 = if executable { U256::one() } else { U256::zero() }; // pad to 32 bytes
```

Comment:
Wouldn't be From trait implementing better?

```rust
impl From<bool> for U256 {
    fn from(value: bool) -> Self {

    }
}
```

Reply:
I prefer explicit expression for conversion between unrelated types instead of hiding it under generic `from`.
This is also important part of the business logic and should be expressed as clear as possible